src/frontend/next/.eslintrc.js

```diff
@@ -0,0 +1,122 @@
+module.exports = {
+  root: true,
+  settings: {
+    react: {
+      version: '16.13',
+    },
+    'import/resolver': {
+      node: {
+        extensions: ['.js', '.jsx', '.ts', '.tsx'],
+      },
+    },
+  },
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module',
+  },
+  extends: [
+    'eslint:recommended',
+    'airbnb-base',
```

Comment:
Isn't `airbnb-base` the one that comes without React plugins?
From `airbnb-base` npm:
>This package provides Airbnb's base JS .eslintrc (without React plugins) as an extensible shared config.

Also, from the `airbnb` npm page:
>If you don't need React, see eslint-config-airbnb-base.

Reply:
Thanks, I'll fix.